src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -17,19 +18,38 @@
     ResultsStrategyFactory,
 )
 from sentry.uptime.detectors.ranking import _get_cluster
+from sentry.uptime.detectors.tasks import set_failed_url
 from sentry.uptime.issue_platform import create_issue_platform_occurrence
-from sentry.uptime.models import ProjectUptimeSubscription, UptimeSubscription
+from sentry.uptime.models import (
+    ProjectUptimeSubscription,
+    ProjectUptimeSubscriptionMode,
+    UptimeSubscription,
+)
+from sentry.uptime.subscriptions.subscriptions import (
+    create_uptime_subscription,
+    delete_project_uptime_subscription,
+    remove_uptime_subscription_if_unused,
+)
 from sentry.utils import metrics
 from sentry.utils.hashlib import md5_text
 
 logger = logging.getLogger(__name__)
 LAST_UPDATE_REDIS_TTL = timedelta(days=7)
+ONBOARDING_FAILURE_THRESHOLD = 3
+ONBOARDING_FAILURE_REDIS_TTL = timedelta(days=2)
+ONBOARDING_MONITOR_PERIOD = timedelta(days=3)
+# How frequently we should run active auto-detected subscriptions
+AUTO_DETECTED_ACTIVE_SUBSCRIPTION_INTERVAL_SECONDS = int(timedelta(minutes=5).total_seconds())
```

Comment:
IMO we should leave this as a timedelta here and convert closer to the usage site. It makes the configuration a bit easier to read, and at the call site it's just as obvious that we're passing seconds.

```suggestion
AUTO_DETECTED_ACTIVE_SUBSCRIPTION_INTERVAL = timedelta(minutes=5)
```

farther down it needs to be

```py
int(AUTO_DETECTED_ACTIVE_SUBSCRIPTION_INTERVAL.total_seconds())
```